### PR TITLE
C++ demos: add .gif files to README

### PR DESCRIPTION
**Pull request by anzhella-pankratova**
add animated GIF files to show how demo output may look like.

- [x] crossroad_barrier_demo
- [x] gaze_estimation_demo
- [x] inteactive_face_detection_demo
- [x] object_detection_demo
- [x] pedestrian_tracker_demo
- [x] security_barrier_camera_demo
- [x] segmentation_demo
- [x] smart_classroom_demo
- [x] python action_recognition
- [x] python whiteboard_inpainting

### CODE REVIEW

File: demos/crossroad_camera_demo/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Crossroad Camera C++ Demo
 
+![](./crossroad_camera.gif)
```
Comment (Wovchena) [inline demos/crossroad_camera_demo/README.md:3]: The usage is valid, but the source video doesn't seem to be representative enough.
`security_barrier_camera.gif` is also run on a bit different video: the demo name suggests that the camera is placed at level of barrier, but not above of passing cars.

@vladimir-dudnik should the gifs be kept?

File: demos/interactive_face_detection_demo/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Interactive Face Detection C++ Demo
 
+![](./interactive_face_detection.gif)
```
Comment (Wovchena) [inline demos/interactive_face_detection_demo/README.md:3]: I think it's better to use all other nets the demo supports like emotions to show all capabilities.

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: demos/crossroad_camera_demo/README.md (hunk 1 above)
Comment (vladimir-dudnik) [reply demos/crossroad_camera_demo/README.md:3]: I think we can use a few subsequent frames from internal validation set, just resize it to lower resolution.
@yssemaev hope there is no legal issues with this, as it should not fall into dataset redistribution case. It is just screenshots of demo execution.

File: demos/interactive_face_detection_demo/README.md (hunk 2 above)
Comment (anzhella-pankratova) [reply demos/interactive_face_detection_demo/README.md:3]: done.

**Comment by vladimir-dudnik** [comment]
@andrew-zaytsev FYI. we are going to add animated GIF to OMZ demo descriptions

File: demos/crossroad_camera_demo/README.md (hunk 1 above)
Comment (yssemaev) [reply demos/crossroad_camera_demo/README.md:3]: Hi, I have no objections if the dataset(s) was approved for Publishing and Distribution cases.

**Review by Wovchena — APPROVED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**